CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.2.2
+
+* Updated several dependencies [#10][]
```

Comment:
I agree on the PR linking. It was requested of me, so I've kept at it.

As for the dependencies, it's just a quick and simple chore to me. They change often, they're already outdated. I'll just close this one.